src/CommunityPatch/Patches/EarlyStoryVisibleTimeoutPatch.cs

```diff
@@ -31,10 +31,10 @@ public sealed class EarlyStoryVisibleTimeoutPatch : IPatch {
 
     private static object SecondPhaseInstance => SecondPhaseInstanceGetter.Invoke(null, null);
 
-    private static readonly int FirstPhaseTimeLimitInYears = ExtractFirstPhaseTimeLimitInYears();
-
     private static readonly FieldInfo FirstPhaseTimeLimitInYearsField = AccessTools.Field(FirstPhaseCampaignBehaviorType, "FirstPhaseTimeLimitInYears");
 
+    private static readonly int? FirstPhaseTimeLimitInYears = ExtractFirstPhaseTimeLimitInYears();
+
```

Comment:
Reversed the order of these because `ExtractFirstPhaseTimeLimitInYears()` depends on `FirstPhaseTimeLimitInYearsField` already being set.

Reply:
Also `FirstPhaseTimeLimitInYears` is nullable to allow the rest of the code to not do anything if we can't retrieve the value.